### ISSUE TITLE
Error : There was a problem loading Inbox

### ISSUE THREAD

**Issue by composteur** [nextcloud/mail#760]
### Steps to reproduce
1. login to my standard account
2. open the mail app
3. open the inbox folder

### Expected behaviour
I should see the complete inbox emails list

### Actual behaviour
I can't see the inbox list but I can see the junk or draft emails list.
I receive a firefox inbox notification !

## Server configuration detail

**Operating system:** Linux info 3.0 #1337 SMP Tue Jan 01 00:00:00 CEST 2000 all GNU/Linux Linux info 3.0 #1337 SMP Tue Jan 01 00:00:00 CEST 2000 all GNU/Linux Linux info 3.0 #1337 SMP Tue Jan 01 00:00:00 CEST 2000 all GNU/Linux Linux info 3.0 #1337 SMP Tue Jan 01 00:00:00 CEST 2000 all GNU/Linux

**Webserver:** Apache (cgi-fcgi)

**Database:** mysql 5.5.58

**PHP version:** 7.1.13
Modules loaded: Core, date, libxml, openssl, pcre, sqlite3, zlib, bcmath, bz2, calendar, ctype, curl, dba, dom, hash, fileinfo, filter, ftp, gd, gettext, SPL, iconv, session, intl, json, mbstring, mcrypt, standard, PDO, mysqlnd, pdo_sqlite, Phar, posix, Reflection, imap, shmop, SimpleXML, soap, pdo_mysql, exif, tidy, tokenizer, wddx, xml, xmlreader, xmlwriter, xsl, zip, mysqli, cgi-fcgi

**Nextcloud version:** 12.0.5 - 12.0.5.3

**Updated from an older Nextcloud/ownCloud or fresh install:** 

**Where did you install Nextcloud from:** unknown

<details><summary>Signing status</summary>

Array
</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - activity: 2.5.2
 - bruteforcesettings: 1.0.3
 - calendar: 1.5.7
 - comments: 1.2.0
 - contacts: 2.0.1
 - dav: 1.3.1
 - federatedfilesharing: 1.2.0
 - federation: 1.2.0
 - files: 1.7.2
 - files_pdfviewer: 1.1.1
 - files_sharing: 1.4.0
 - files_texteditor: 2.4.1
 - files_trashbin: 1.2.0
 - files_versions: 1.5.0
 - files_videoplayer: 1.1.0
 - firstrunwizard: 2.1
 - gallery: 17.0.0
 - issuetemplate: 0.3.0
 - logreader: 2.0.0
 - lookup_server_connector: 1.0.0
 - mail: 0.7.9
 - news: 11.0.5
 - nextcloud_announcements: 1.1
 - notifications: 2.0.0
 - oauth2: 1.0.5
 - password_policy: 1.2.2
 - provisioning_api: 1.2.0
 - serverinfo: 1.2.0
 - sharebymail: 1.2.0
 - survey_client: 1.0.0
 - systemtags: 1.2.0
 - theming: 1.3.0
 - twofactor_backupcodes: 1.1.1
 - updatenotification: 1.2.0
 - workflowengine: 1.2.0
Disabled:
 - admin_audit
 - encryption
 - files_external
 - richdocuments
 - user_external
 - user_ldap

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "instanceid": "oc6mbda98ctk",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "stratus.kakinoki.fr"
    ],
    "datadirectory": "\/homepages\/23\/d144621702\/htdocs\/kakinoki\/nextcloud\/data",
    "overwrite.cli.url": "http:\/\/stratus.kakinoki.fr",
    "dbtype": "mysql",
    "version": "12.0.5.3",
    "dbname": "db620631510",
    "dbhost": "db620631510.db.1and1.com",
    "dbport": "",
    "dbtableprefix": "oc_",
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "logtimezone": "UTC",
    "installed": true,
    "app.mail.transport": "php-mail",
    "app.mail.imaplog.enabled": "true",
    "mail_smtptimeout": "30",
    "maintenance": false,
    "theme": "",
    "loglevel": 2,
    "updater.secret": "***REMOVED SENSITIVE VALUE***"
}
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

<details><summary>LDAP configuration (delete this part if not used)</summary>

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
</details>

## Client configuration

**Browser:** Firefox 0.0 

**Operating system:** 

## Logs

<details><summary>Browser log</summary>

```
Insert your webserver log here 
```
</details>

<details><summary>Nextcloud log</summary>

```
Insert your Nextcloud log here
```
</details>

<details><summary>Browser log</summary>

Insert your browser log here, this could for example include:

	a) The javascript console log
	b) The network log
	c) ...
</details>

**Comment by ChristophWurst** [comment]
> Nextcloud log
> 
> Insert your Nextcloud log here

Please post your log entries when this error occurs. Additionally, enable debug mode and post your horde_imap.log.

You can find the full issue template here: https://raw.githubusercontent.com/nextcloud/mail/master/.github/issue_template.md - would be great if you could add the missing sections. Thanks.

**Comment by composteur** [comment]
I'm so sorry to have disturbed you. I recently look at the https security indicator in firefox. It was red, I don't know why. So I just made a "firefox repair" and it become green. After logon to nextcloud, all work fine with the emails. I can't explain the reason why the security have been compromised. I use a Linux distro with standard repositories. I changed all the mail passwords, just in case. Thanks for your attention and quick answer !
